__tests__/app.test.js

```diff
@@ -9,6 +9,14 @@ beforeEach(() => seed(topicData))
 
 afterAll(() => db.end)
 
+describe.only('/api', () => {
```

Comment:
if you require in the endpoints.json file you can then compare to see if the response body is the same as the required file rather then just checking for length